scripts/deploy-templates.ts

```diff
@@ -0,0 +1,116 @@
+import { createTemplate, deriveAirnodeXpub, deriveSponsorWalletAddress } from '@api3/airnode-admin';
+import { AirnodeRrp } from '@api3/airnode-protocol';
+import { ethers, Wallet } from 'ethers';
+import * as node from '@api3/airnode-node';
+import {
+  cliPrint,
+  deriveKeeperSponsorWallet,
+  getDeployedContract,
+  getProvider,
+  readIntegrationInfo,
+  runAndHandleErrors,
+} from '../src';
+
+const main = async () => {
+  const provider = getProvider();
+  const integrationInfo = await readIntegrationInfo();
+  const airnodeRootWallet = ethers.Wallet.fromMnemonic(integrationInfo.mnemonic).connect(provider);
+
+  const airnodeRrpInstance = (
+    (await getDeployedContract('@api3/airnode-protocol/contracts/rrp/AirnodeRrp.sol')) as AirnodeRrp
+  ).connect(airnodeRootWallet);
+  cliPrint.info(`The Deployed AirnodeRrp Contract's Address: ${airnodeRrpInstance.address}`);
+
+  const pairs = ['eth_dai', 'dai_usd', 'usdc_usd', 'usdt_usd', 'usdt_eth', 'usdc_eth', 'eth_btc', 'eth_usd', 'btc_usd'];
+
+  const tickerTemplate = {
+    airnode: airnodeRootWallet.address,
+    endpointId: '0x3718f9f03845bab74e16647b30034960eab7cb5fcff451651f3624fff6974026',
```

Comment:
Great idea, definitely.

Reply:
Interestingly, deriving the endpointId using the admin package/AbiEncoder+keccak does not result in the same value used in the Amberdata `config.json` file... so the person who generated the file must have done something non-standard.

Reply:
Btw, well done on this Santiago, you helped discover that the endpointIds had been incorrectly generated :)

Reply:
Thanks Aaron and funny enough, I did too run the `derive-endpoint-id` command and thought I was doing something wrong because I did not get the expected result 😄